Small changes (#6590)
magento/devdocs: src/guides/v2.3/javascript-dev-guide/javascript/requirejs.md

}
```

{:.bs-callout-tip}

> The `map` configuration can also be used to override a JS module with a custom JS module. More information can be found at [Custom JS component]({{ page.baseurl }}/javascript-dev-guide/javascript/custom_js.html#js_replace)

Reviewer: ```suggestion
You can also use the `map` configuration to override a JS module with a custom JS module. See [Custom JS component]({{ page.baseurl }}/javascript-dev-guide/javascript/custom_js.html#js_replace).
```
